plugins/woocommerce/templates/loop/result-count.php

```diff
@@ -14,14 +14,14 @@
  *
  * @see         https://woocommerce.com/document/template-structure/
  * @package     WooCommerce\Templates
- * @version     9.4.0
+ * @version     9.8.0
```

Comment:
```suggestion
 * @version     9.9.0
```

As the [WooCommerce 9.8 Feature Freeze](https://developer.woocommerce.com/release-calendar/) took place on March 3, 2025, the version should be bumped to `9.9.0`.

Reply:
Thanks, @nielslange 
I just updated the template version.